src/systems/buoyancy/Buoyancy.cc

```diff
@@ -88,6 +89,21 @@ class gz::sim::systems::BuoyancyPrivate
   void GradedFluidDensity(
     const math::Pose3d &_pose, const T &_shape, const math::Vector3d &_gravity);
 
+  /// \brief Check and populate new buoyancy components
+  /// \param[in] _ecm The Entity compontent manager.
```

Comment:
@arjo129 typo:
```suggestion
  /// \param[in] _ecm The Entity component manager.
```
Same below.